cerebrium/fine-tuning/diffusion-models/dataset.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: "Dataset Curation"
+description: "Guide to curating a dataset for success when fine-tuning your diffusion model on Cerebrium."
+---
+
+<Note>
+  Cerebrium's fine-tuning functionality is in public beta and we are adding
+  more functionality each week! If there are any issues or if you have an urgent requirement, please reach
+  out to [support](mailto\:support@cerebrium.ai) 
```

Comment:
I don't think it should have that slash in there? 🤔

Reply:
Thanks. Didn't check that one. There were a couple others like that too. Good save.